tpu-provisioner/internal/cloud/gke_test.go

```diff
@@ -227,14 +232,102 @@ func TestPodToNodePoolName(t *testing.T) {
 	}
 }
 
-func TestNodePoolForPod(t *testing.T) {
+func buildPod(additionalLabels map[string]string, additionalAnnotations map[string]string, selector map[string]string, podSpec *v1.PodSpec) *corev1.Pod {
```

Comment:
In general, the rest of the files put helper funcs below the funcs that call them, can you rearrange to follow?

Reply:
done